pkg/network/master/master.go

```diff
@@ -151,11 +156,21 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, networkClient networkc
 		return err
 	}
 
+	if err := master.initSubSystems(); err != nil {
+		return err
+	}
 	go master.startSubSystems(networkConfig.NetworkPluginName)
 
 	return nil
 }
 
+func (master *OsdnMaster) initSubSystems() error {
```

Comment:
There's no reason this has to be a separate function, is there? You could just as easily call initSubnetMaster() at the start of startSubSystems().

Actually is there even any reason left at this point for separate init and start methods? It's true that we do that in some other files but that's usually because the start method does things we don't want to do from unit tests. But there are no unit tests of these files (and the init method does things we probably wouldn't want done from unit tests as well anyway).

Same comment applies to vnid code